server/config/middleware.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+var userRoutes = require('../users/userRoutes.js');
+var messageRoutes = require('../messages/messageRoutes.js');
+var contactRoutes = require('../contacts/contactRoutes.js');
+
+module.exports = function (app, express) {
+
+  // instantiate express Routers for User, Message, and Contact 
+  // see: http://expressjs.com/guide/routing.html
+  var userRouter = express.Router();
+  var messageRouter = express.Router();
+  var contactRouter = express.Router();
+
```

Comment:
@onelovelyname yeah, we would still need express.static because the server needs to actually serve the client/index.html of the SPA. The Angular router in client/index.html will then serve client/app/index.html as the template for the landing page.